tests/decorators/tests.py

```diff
@@ -7,12 +8,14 @@
 )
 from django.http import HttpRequest, HttpResponse, HttpResponseNotAllowed
 from django.middleware.clickjacking import XFrameOptionsMiddleware
-from django.test import SimpleTestCase
+from django.template import Context, Template
```

Comment:
I think it would be much preferred if the tests didn't introduce a dependency on the template system.